observer/stream.go

```diff
@@ -100,7 +101,10 @@ func (s *Stream) loadBlock(c chan<- *blockatlas.Block, num int64) {
 	// Not strictly correct nor avoids race conditions
 	// But good enough
 	newNum := atomic.AddInt64(&s.blockNumber, 1)
-	err = s.Tracker.SetBlockNumber(s.coin, newNum)
```

Comment:
Hmm  looks like you have now removed the actual persisting of block height though.

Reply:
I only commited your suggestion. Will review

Reply:
Done